docs/md/dev-process.md

```diff
@@ -1,26 +1,22 @@
 # Development Process
 
-This is a development process
-we are tying to standardize
-within the team,
-and encourage ourselves
-to follow in most cases.
+This is a development process we are tying to standardize within the team and
+encourage ourselves to follow in most cases.
 
 ## The Swagger page
 
-WFL ships with a swaggger UI that documents all of the
-available endpoints. It's available at path `/swagger`, e.g.
+WFL ships with a swaggger UI that documents all of the available endpoints.
+It's available at path `/swagger`, e.g.
 https://dev-wfl.gotc-dev.broadinstitute.org/swagger
```

Comment:
I tripped on this when setting up WFL: because Swagger is bundled with the UI and not the API we can't hit the endpoint directly.  Ultimately I know we wish to fix this but in the meantime, could update the docs to indicate that we first must go to WFL UI directly and log in before accessing Swagger.

Reply:
Added a clarifying comment